src/LogInPage.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+
+export default class LogInPage extends Component {
+  render () {
+    return (
+      <div id='logIn' >
+        <form id='logInInputs' onSubmit={this.props.logIn}>
+          <label htmlFor='email'>Email</label>
+          <input name='email' id='email' type='text' placeholder='Enter text' onChange={this.props.handleChange} />
```

Comment:
`placeholder='Enter email address'`

also: `type='email'` instead of `type='text'` to get email validation for free from HTML5.